src/compute/src/extensions/arrange.rs

```diff
@@ -282,7 +279,7 @@ where
         log_arrangement_size_inner(self, |trace| {
             let (mut size, mut capacity, mut allocations) = (0, 0, 0);
             let mut callback = |siz, cap| {
-                allocations += 1;
+                allocations += (cap > 0).then_some(1).unwrap_or(0);
```

Comment:
```suggestion
                allocations += (cap > 0).into();
```

Reply:
`into` doesn't actually work, and the `usize::from` variant seems cleaner:
```
error[E0282]: type annotations needed
   --> src/compute/src/extensions/arrange.rs:282:42
    |
282 |                 allocations += (cap > 0).into();
    |                                          ^^^^
    |
help: try using a fully qualified path to specify the expected types
    |
282 |                 allocations += <bool as Into<T>>::into((cap > 0));
    |                                ++++++++++++++++++++++++         ~
```